README.md

```diff
@@ -1 +1,33 @@
-# Hello-world
\ No newline at end of file
+
+
+# Set up a "Hello Microverse" project
+
+## Description 0f the project.
+In this project, i have set up a "Hello world" repository.My goal here is to master all of the tools and best practices GitHub flow,linters. I have to be familiar with them since i will be using them in all Microverse projects and most likely in my future job as well, so it is important to know them!
+
+
+## Built With
+- HTML
+- CSS
+- linter for HTML and CSS
+- Node Page modules(NPM)
+
+## Authors
+
+👤 **Balikuddembe Emmanuel**
+
+- GitHub: [@Balikuddembe](https://github.com/Balikuddembe)
+- Twitter: [@EmmanuelBaliku1](https://twitter.com/EmmanuelBaliku1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuel-balikuddembe-763765b2/)
+
+
+
+Contributions to this repo are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
```

Comment:
- [ optional ] Please try to  Update the README file , it should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo.